.github/workflows/benchmark.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         llvm: ["7",  "8", "9", "10", "11", "12", "13"] #, "14"]
         build: ["Release", "Debug"] # "RelWithDebInfo"
         os: [self-hosted]
-    timeout-minutes: 120 
+    timeout-minutes: 300 
```

Comment:
We can probably split bench forward/reverse as two separate things (and I should start more CI)

Reply:
I would disable the fwd tests by default. At some point we need to decide on a subset of test cases that run in reasonable time.